src/Maker/MakeResetPassword.php

```diff
@@ -236,10 +237,24 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             EntityManagerInterface::class,
         ];
 
+        // Namespace for ResetPasswordExceptionInterface was imported above
+        $problemValidateMessageOrConstant = \defined('SymfonyCasts\Bundle\ResetPassword\Exception\ResetPasswordExceptionInterface::MESSAGE_PROBLEM_VALIDATE')
+            ? 'ResetPasswordExceptionInterface::MESSAGE_PROBLEM_VALIDATE'
+            : "'There was a problem validating your password reset request'";
+        $problemHandleMessageOrConstant = \defined('SymfonyCasts\Bundle\ResetPassword\Exception\ResetPasswordExceptionInterface::MESSAGE_PROBLEM_VALIDATE')
+            ? 'ResetPasswordExceptionInterface::MESSAGE_PROBLEM_HANDLE'
+            : "'There was a problem handling your password reset request'";
+
+        $optionalTranslatorParams = [];
+        if ($isTranslatorAvailable = interface_exists(TranslatorInterface::class)) {
+            $optionalTranslatorParams['translator_class_details'] = $generator->createClassNameDetails(TranslatorInterface::class, '\\');
```

Comment:
Fairly speaking, I'm not sure we need to create class name details for translator as well. I just did the same way `password_hasher_class_details` is implemented. Should we just hardcode translator class instead?

Reply:
Thank you for sharing some thoughts about it! I made this change, the logic would be simpler this way unless we really need to use some benefits of that `createClassNameDetails()`